README.md

```diff
@@ -85,7 +85,7 @@ following ways (or however else they see fit):
 
 ### Role Changes
 
-*   Core owners may choose to step down and leave the organization at any time.
+*   Core owners may choose to step down and become core collaborators, or choose leave to leave the organization entirely, at any time.
```

Comment:
Minor typo here: "or choose leave to leave" rather than "or choose to leave"

Reply:
Oops, thanks.